src/pages/admin/membership-builder.vue

```diff
@@ -0,0 +1,46 @@
+<route lang="yaml">
+name: adminMembershipBuilder
+meta:
+  pageTitle: membershipBuilder.title
+  role: admin
+</route>
+
+<template>
+  <PageTitle :title="t('membershipBuilder.title')" border />
```

Comment:
As per the design spec, page titles with horizontal tabs don't have a border. You can remove `border` attribute :)